interoperability.md

```diff
@@ -0,0 +1,39 @@
+# Assess interoperability of Presentation API implementations
+
+This document discusses the interoperability of potential Presentation API implementations in particular interoperability
+between user agents in case of [2-UA implementations][2ua] and between user agents and presentation displays in case of [1-UA implementations][1ua].
+For this, state of the art technologies and standards that may be useful to implement the Presentation API will be considered.
+
+## Interoperability between user agents (2-UA)
+
+The following diagram shows an overview of the main components for the 2-UA implementation. Each of the user agents implements
+part of the Presentation API (for controlling browsing context or presenting browsing context). To ensure interoperability
+between user agents running on controller devices and user agents running on presentation devices the following aspects should be considered:
+
+* **Discovery**: to discover presentation devices, the controlling user agent should use the same technology as the presenting
+user agent uses to advertise itself. **Potential technologies**: SSDP, mDNS or BLE (BLE Beacon)
+* **Launch**: after discovery, the controlling user agent should know how to launch a new presentation or join an existing one.
+Common message formats should be also used. **Potential technologies**: DIAL (uses SSDP for discovery), Google Cast (part of it).
+* **Communication**: also for communication both user agents should use same technologies/protocols to ensure interoperability.
+**Potential technologies**: WebSockets, WebRTC, Raw Socket, Google Cast (part of it)
+* **Signaling**: common language and message formats (e.g. using JSON or XML) should be used to exchange signaling information between the two user agents.
+
```

Comment:
I would add security to implemenent authentication of the display and privacy of the presentation messaging.